src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/PreferAsSpanOverSubstring.cs

```diff
@@ -33,7 +33,7 @@ public sealed class PreferAsSpanOverSubstring : DiagnosticAnalyzer
             isPortedFxCopRule: false,
             isDataflowRule: false);
 
-        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(Rule);
+        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get; } = ImmutableArray.Create(Rule);
```

Comment:
Not related to this PR, you may want to move this over to your other PR.

Reply:
@Evangelink Indeed. I did `git stash apply` on the wrong branch 😄